lib/bamboo/adapters/mandrill_helper.ex

```diff
@@ -48,4 +48,20 @@ defmodule Bamboo.MandrillHelper do
   def tag(email, tags) do
     put_param(email, "tags", List.wrap(tags))
   end
+
+  @doc """
+  Setup Mandrill to send using a named template with template content. Use this
```

Comment:
Usually the first sentence of docs should be the summary of the function. So maybe something like

```elixir
Send emails using Mandrill's template API

...rest of the description here
```